src/middlewared/middlewared/plugins/ldap.py

```diff
@@ -28,6 +34,87 @@ class SSL(enum.Enum):
     USETLS = 'START_TLS'
 
 
+class NlscdConst(enum.Enum):
+    NSLCD_CONF_PATH = '''/usr/local/etc/nslcd.conf'''
+    NSLCD_PIDFILE = '''/var/run/nslcd.pid'''
+    NSLCD_SOCKET = '''/var/run/nslcd/nslcd.ctl'''
```

Comment:
Why triple quote? thats odd : P

Reply:
It's in sample tools in nss-pam-ldapd. It bugs me too so I'll remove it.